src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString.split(';');
+  const result = {};
+
+  properties.forEach((property) => {
+    const objProperty = property.trim();
+
+    const [key, value] = objProperty.split(':');
```

Comment:
Ensure that the code is robust against potential issues with the input string. For example, if there are multiple colons ':' within a declaration, this split would not work as intended. Consider using a regular expression or limiting the split to the first colon found.